core/src/cluster_info_vote_listener.rs

```diff
@@ -881,7 +884,7 @@ mod tests {
             &votes_receiver,
             &vote_tracker,
             &bank3,
-            &subscriptions,
+            &Arc::try_unwrap(subscriptions.clone()).ok(),
```

Comment:
These tests will get much simpler too, since you can just pass `subscriptions.as_deref()`